README.md

```diff
@@ -1,15 +1,15 @@
 geosite-framework-build
 =======================
 
-Build script and optional auto-installer for GeositeFramework regions.
+Build script for GeositeFramework regions.
 
 ### Usage
 
 ```bash
 $ python build.py -h
```

Comment:
Shouldn't the required arguments be listed too? `[REGION_REPO or REGION_CONFIG FILE] [optional: REGION_REPO, default: CoastalResilienceNetwork]` or something to that extent but clearer.

Reply:
The region repo is there, but it's named `source` and is listed with more detail under the "positional arguments" header. This is the output from running `python build.py -h`.

Reply:
This was a bit out-of-date, so I updated it.

Reply:
ohh, i see `source [org]` listed at the end of the `usage` block now. That's a little confusing it's not at the beginning of the args list like `build.py -h source [org] [--region-branch...]`

Reply:
I agree with that, but the help output is autogenerated. I looked into the argparser module and messed with some options (like explicitly marking things as required), but none of them moved around the argument positions. I think keeping the help output of the readme in sync with the output of the tool is easiest for future updates. Also, I think that all of the usage examples in the README sections that follow this one make things clearer.

Reply:
Also we may revisit this after TNC gets a chance to test it out.